charts/postgres-operator-ui/values.yaml

```diff
@@ -8,7 +8,7 @@ replicaCount: 1
 image:
   registry: registry.opensource.zalan.do
   repository: acid/postgres-operator-ui
-  tag: v1.8.2
+  tag: v1.9.0
```

Comment:
@FxKu do you mind that `master` branch should contains `latest` tag and specific version tag should be present inside the chart archive only. Otherwise is not possible to install chart from the current master in the development (with some medication manifests values).
wdyt about it?

Reply:
Yes, indeed. We could always have the `latest` tag and `imagePullPolicy: Always` on the `master` and create release script to automate this operation.

Reply:
Keep in mind! The default `imagePullPolicy` always pulls the `:latest` tag! As such, it should not be specified for this usecase.

https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting

Reply:
@katlol I know about it, thank you. But for this case, it's better to specified it to simplify release scripts for helm chart if this approach could be implemented.